Bottom: f0c180d833d20dbe1c788452b8e743b3b0ef6815
Top:    b4dc778adb0078acdc7b5c17ef2d6ec59c167da1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 22:23:29 +0100

action.c: Remove dead code, IgnoreMotionEvents


---

diff --git a/src/action.c b/src/action.c
index d892429..6a339bb 100644
--- a/src/action.c
+++ b/src/action.c
@@ -323,7 +323,6 @@ static int defer_updates = 0;
 static int defer_needs_update = 0;
 
 static Cardinal polyIndex = 0;
-static bool IgnoreMotionEvents = false;
 static bool saved_mode = false;
 #ifdef HAVE_LIBSTROKE
 static bool mid_stroke = false;
@@ -2153,17 +2152,12 @@ EventMoveCrosshair (int ev_x, int ev_y)
     }
 #endif /* HAVE_LIBSTROKE */
   /* ignore events that are caused by ActionMovePointer */
-  if (!IgnoreMotionEvents)
+  if (MoveCrosshairAbsolute (ev_x, ev_y))
     {
-      if (MoveCrosshairAbsolute (ev_x, ev_y))
-	{
-	  /* update object position and cursor location */
-	  AdjustAttachedObjects ();
-	  notify_crosshair_change (true);
-	}
+      /* update object position and cursor location */
+      AdjustAttachedObjects ();
+      notify_crosshair_change (true);
     }
-  else
-    IgnoreMotionEvents = false;
 }
 
 /* --------------------------------------------------------------------------- */
